package.json

```diff
@@ -13,10 +13,13 @@
     "target/src"
   ],
   "dependencies": {
-    "fp-ts": "^0.6.2",
-    "io-ts": "^0.8.2"
+    "fp-ts": "^0.6.8"
+  },
+  "peerDependencies": {
+    "io-ts": "^0.9.7"
   },
   "devDependencies": {
+    "io-ts": "^0.9.7",
```

Comment:
Does this need to be a dev dependency as well as a peer dependency?

I guess so, because peer dependencies won't be installed when you're developing the package locally? Is that correct?